mxcube3/ui/actions/general.js

```diff
@@ -152,11 +176,15 @@ export function getInitialStatus() {
     const pchains = [
```

Comment:
As I am making the review, I looked at all those calls... I think we should do some refactoring because over the internet an AJAX call is never less than 100 ms (more like 200 ms in fact) and this initialization will take too much time in my opinion. The idea could be to group some of those ; what about having `motorLimits` with `motors`, and `acqParametersLimits` + `dcParameters`  with `beamline(Limits/Setup)`, or `dataPath` with  `queue` (this is only suggestions).